### ISSUE TITLE
Nipple thickening

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10111]
**Preferred term label:**
Nipple thickening

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0004404

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lobular breast carcinoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
does this need to be differentiated from 
Asymmetric thickening of the nipple-areolar complex
?

**Comment by MickeySegal** [comment]
The other finding stresses the asymmetry.  This stresses thickening with no comment on symmetry.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000629
- label: Nipple thickening
- parents: [Abnormal nipple morphology (HP:0004404)](https://hpo.jax.org/app/browse/term/HP:0004404)
- pmid: 
- def: Thickening of the skin around the nipple.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10111
- synonyms:
